devops/eslint-config/index.js

```diff
@@ -0,0 +1,50 @@
+// This config is used globally at the root of the repo, so it should be as thin
+// as possible with rules that we absolutely require across all projects.
+module.exports = {
+  env: {
+    es6: true,
+  },
+  globals: {
+    Atomics: 'readonly',
+    SharedArrayBuffer: 'readonly',
+  },
+  // We are relying on version that comes with @polkadot/dev
+  // Newest version is breaking pioneer!
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 2019,
+    sourceType: 'module',
+  },
+  extends: [
+    'standard',
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:react/recommended',
+    // this is only in newer versions of eslint-plugin-react-hooks
+    // 'plugin:react-hooks/recommended',
+
+    // Turns off all rules that are unnecessary or might conflict with Prettier.
+    // Allows us to do formatting separately from linting.
+    'prettier',
+    'prettier/@typescript-eslint',
+    'prettier/react',
+    'prettier/standard',
+    // To Display prettier errors as ESLint errors, enable the following configuration,
+    // And make sure it is the last configuration in the extends array.
+    'plugin:prettier/recommended',
```

Comment:
Just noticed, in example config by Prettier this is actually above all other `prettier` rules (e.g. `prettier/react`, etc.)

Reply:
The docs say this line:
> - Enables eslint-plugin-prettier.
> - Sets the prettier/prettier rule to "error".
> - Extends the eslint-config-prettier configuration.

So it may be a replacement for `prettier` above

Reply:
Yes I saw the example. I was basing the ordering on this [article](https://www.robertcooper.me/using-eslint-and-prettier-in-a-typescript-project)

The way I understand it is that `eslint-plugin-prettier` actually runs prettier itself and reports back the formatting errors based on your prettierrc config. So it seems natural to put it last. Perhaps the order of the plugin itself is not important because its not actually modifying any rules just executing and that happens after all the rules have been configured. I did test the behavior on both the command line and in the IDE and I got the expected results. But I guess it makes sense to follow the maintainers documentation :)

Reply:
I asked out of curiosity https://github.com/prettier/eslint-plugin-prettier/issues/314

Reply:
Okay so I got an answer which makes sense as far as rules go, but didn't satisfy my curiosity about when prettier runs.
So I did a bit of debugging and can confirm that prettier runs (the `create()` method is called on the plugin) after all the rules in the extends array processed.

Reply:
Awesome, good to know